cmd/snap/cmd_snap_op.go

```diff
@@ -73,6 +73,10 @@ The remove command removes the named snap instance from the system.
 By default all the snap revisions are removed, including their data and the
 common data directory. When a --revision option is passed only the specified
 revision is removed.
+
+Unless automatic snapshots are disabled, a snapshot of all data for the snap is 
+saved upon removable and available for future restoration with snap restore. The
```

Comment:
small comment: `saved upon removal, which is then available for future restoration with snap restore.`

Reply:
Done, thanks